### PR TITLE
Added new changes to take failed screenshots

### PR DESCRIPTION
**Pull request by vinodkumarsharma276**

### CODE REVIEW

File: src/StoryWrightProcessor/PlayWrightExecutor.ts [hunk 1]
```diff
@@ -143,6 +150,7 @@ export class PlayWrightExecutor {
 
   private hover = async (selector: string) => {
     try {
+      console.log(`selector: ${selector}`);
```
Comment (p01) [inline src/StoryWrightProcessor/PlayWrightExecutor.ts:153]: Let's make sure to remove the `console.log()` before merging

File: src/StoryWrightProcessor/PlayWrightExecutor.ts [hunk 2]
```diff
@@ -143,6 +150,7 @@ export class PlayWrightExecutor {
...
       const element = await this.page.$(selector);
```
Comment (p01) [inline src/StoryWrightProcessor/PlayWrightExecutor.ts:154]: Shouldn't we curate the selector here in all other places as well ?

File: src/StoryWrightProcessor/PlayWrightExecutor.ts [hunk 3]
```diff
@@ -201,4 +209,34 @@ export class PlayWrightExecutor {
...
+
+    while(selector.indexOf("[") > -1 && selector.indexOf("=") > -1){
+      let temp = selector.substring(selector.indexOf("=") + 1, selector.indexOf("]"));
+
+      if(!(temp.charAt(0) == '"' || temp.charAt(0) == '\'')){
+        temp = '"' + temp + '"';
+      }
+      newSelector += temp + "]";
+      selector = selector.substring(selector.indexOf("]")+1, selector.length);
+    }
```
Comment (p01) [inline src/StoryWrightProcessor/PlayWrightExecutor.ts:235]: Could you please add some comments showing the type of selectors we are looking for and how we change them.

File: src/StoryWrightProcessor/StoryWrightProcessor.ts [hunk 4]
```diff
@@ -66,7 +66,7 @@ export class StoryWrightProcessor {
             itemsForBatch.map(async (story: object) => {
               const id: string = story["id"];
               // Set story category and name as prefix for screenshot name.
-              const ssNamePrefix = `${story["kind"]}.${story["name"]}`;
+              const ssNamePrefix = `${story["kind"]}.${story["name"]}`.replace("/", "-"); //INFO: '/' in screenshot name creates a folder in screenshot location. Replacing with '-'
```
Comment (p01) [inline src/StoryWrightProcessor/StoryWrightProcessor.ts:69]: Let's do this for `"\\"` as well.

File: src/StoryWrightProcessor/PlayWrightExecutor.ts [hunk 5]
```diff
@@ -97,6 +101,9 @@ export class PlayWrightExecutor {
 
   private click = async (selector: string) => {
     try {
+      console.log("selector: ", selector);
+      selector = this.curateSelector(selector);
```
[the remark this answers is not in the document]
Comment (vinodkumarsharma276) [reply src/StoryWrightProcessor/PlayWrightExecutor.ts:108]: Removed from all places.

File: src/StoryWrightProcessor/PlayWrightExecutor.ts (hunk 2 above)
Comment (vinodkumarsharma276) [reply src/StoryWrightProcessor/PlayWrightExecutor.ts:157]: Added curate() at all required places.

**Pull request merged**